platform/machine/packet/console.go

```diff
@@ -38,7 +39,10 @@ func (c *console) Write(p []byte) (int, error) {
 }
 
 func (c *console) Close() error {
-	close(c.done)
+	if !c.closed {
```

Comment:
see https://github.com/flatcar-linux/mantle/pull/268/files#r770517411 on why this here is not really needed